ci-operator/step-registry/stackrox/collector/update-support-packages/teardown/stackrox-collector-update-support-packages-teardown-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+source .openshift-ci/jobs/update-support-packages/env.sh
+
+.openshift-ci/jobs/integration-tests/gcloud-init.sh
+
+exec .openshift-ci/jobs/integration-tests/teardown-vm.sh
```

Comment:
Just out of curiosity, why do we `exec` here instead of just running the script?

Reply:
Integration tests do this as well. There doesn't seem to be a reason.

Reply:
I have removed the exec.